### PR TITLE
Saved query parameters

### PR DESCRIPTION
**Pull request by rjmac**

### CODE REVIEW

File: soql-standalone-parser/src/main/scala/com/socrata/soql/parsing/RecursiveDescentParser.scala [hunk 1]
```diff
@@ -1159,6 +1159,38 @@ abstract class RecursiveDescentParser(parameters: AbstractParser.Parameters = Ab
...
+              case _ =>
+                fail(reader.rest.rest, AStringLiteral)
+            }
+          case _ =>
+            fail(reader.rest, COMMA())
+        }
+      case tokens.StringLiteral(varName) =>
+        reader.rest.first match {
+          case RPAREN() =>
+            ParseResult(reader.rest.rest, Hole.SavedQuery(HoleName(varName), None)(pos))
```
Comment (sjzavaleta) [inline soql-standalone-parser/src/main/scala/com/socrata/soql/parsing/RecursiveDescentParser.scala:1185]: I thought we might be adding the inferred table name here, does that happen somewhere else?

File: soql-analyzer/src/test/scala/com/socrata/soql/parsing/ParserTest.scala [hunk 2]
```diff
@@ -176,6 +179,11 @@ class ParserTest extends WordSpec with MustMatchers {
       x.selection.expressions.head.expression.toString must be (largeNumber)
     }
 
+    "accept the various parameter synaxes" in {
```
Comment (rlvoyer) [inline soql-analyzer/src/test/scala/com/socrata/soql/parsing/ParserTest.scala:182]: nit: typo

File: soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLTypeInfo.scala [hunk 3]
```diff
@@ -1,5 +1,6 @@
 package com.socrata.soql.functions
 
+import com.socrata.soql.ast.Hole
```
Comment (rlvoyer) [inline soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLTypeInfo.scala:3]: Not yet used?

Comment (rjmac) [reply soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLTypeInfo.scala:3]: Leftover from a time when the holes were typed.

**Review by sjzavaleta — APPROVED** [review]

**Pull request merged**